PubNubSwiftConsole/Classes/UI/CollectionView/Cells/SubscribeStatusCollectionViewCell.swift

```diff
@@ -31,15 +33,21 @@ struct SubscribeStatus: SubscribeStatusItem {
     let creationDate: NSDate
     let statusCode: Int
     var timeToken: NSNumber?
-    init(itemType: ItemType, status: PNStatus) {
+    var channel: [String]?
+    var channelGroup: [String]?
+    var client: PubNub
+    init(itemType: ItemType, status: PNStatus, client: PubNub) {
         self.itemType = itemType
         self.category = status.stringifiedCategory()
         self.operation = status.stringifiedOperation()
         self.creationDate = NSDate()
         self.statusCode = status.statusCode
         if let subscribeStatus = status as? PNSubscribeStatus {
             self.timeToken = subscribeStatus.data.timetoken
+            self.channel = subscribeStatus.subscribedChannels
+            self.channelGroup = subscribeStatus.subscribedChannelGroups
         }
+        self.client = client
```

Comment:
this means that if the cell is refreshed it'll show the data of the client at the moment of refresh, not from the actual status